[2024-07-08T16:33:09.761+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T16:33:09.802+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_store.dds-load manual__2024-07-08T16:18:08.118541+00:00 [queued]>
[2024-07-08T16:33:09.812+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_store.dds-load manual__2024-07-08T16:18:08.118541+00:00 [queued]>
[2024-07-08T16:33:09.813+0000] {taskinstance.py:2306} INFO - Starting attempt 6 of 6
[2024-07-08T16:33:09.829+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): dds-load> on 2024-07-08 16:18:08.118541+00:00
[2024-07-08T16:33:09.836+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=761) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T16:33:09.838+0000] {standard_task_runner.py:63} INFO - Started process 763 to run task
[2024-07-08T16:33:09.838+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_store', 'dds-load', 'manual__2024-07-08T16:18:08.118541+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/etl_store.py', '--cfg-path', '/tmp/tmpt5az9rnb']
[2024-07-08T16:33:09.841+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask dds-load
[2024-07-08T16:33:09.904+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_store.dds-load manual__2024-07-08T16:18:08.118541+00:00 [running]> on host a58d746d8a30
[2024-07-08T16:33:10.012+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_store' AIRFLOW_CTX_TASK_ID='dds-load' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T16:18:08.118541+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-08T16:18:08.118541+00:00'
[2024-07-08T16:33:10.015+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T16:33:10.027+0000] {sql.py:276} INFO - Executing: 
    insert into dds.fact_sales (invoice_id, branch, city, customer_type, gender, product_line, unit_price, quantity, "tax_5%",
        total, date, time, payment, cogs, gross_margin_percentage, gross_income, rating)
        (select distinct 
            invoice_id, branch, city, customer_type, gender, 
            product_line, unit_price, quantity, "tax_5%", total, date::date,
            time, payment, cogs, gross_margin_percentage, gross_income, rating 
        from nds.fact_sales where invoice_id not in (select distinct invoice_id FROM dds.fact_sales));
[2024-07-08T16:33:10.041+0000] {base.py:84} INFO - Using connection ID 'postgres-netology' for task execution.
[2024-07-08T16:33:10.052+0000] {base.py:84} INFO - Using connection ID 'postgres-netology' for task execution.
[2024-07-08T16:33:10.066+0000] {sql.py:487} INFO - Running statement: 
    insert into dds.fact_sales (invoice_id, branch, city, customer_type, gender, product_line, unit_price, quantity, "tax_5%",
        total, date, time, payment, cogs, gross_margin_percentage, gross_income, rating)
        (select distinct 
            invoice_id, branch, city, customer_type, gender, 
            product_line, unit_price, quantity, "tax_5%", total, date::date,
            time, payment, cogs, gross_margin_percentage, gross_income, rating 
        from nds.fact_sales where invoice_id not in (select distinct invoice_id FROM dds.fact_sales));, parameters: None
[2024-07-08T16:33:10.127+0000] {sql.py:496} INFO - Rows affected: 1000
[2024-07-08T16:33:10.134+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T16:33:10.147+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_store, task_id=dds-load, run_id=manual__2024-07-08T16:18:08.118541+00:00, execution_date=20240708T161808, start_date=20240708T163309, end_date=20240708T163310
[2024-07-08T16:33:10.173+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T16:33:10.193+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-08T16:33:10.197+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
